test/WebSites/AutofacWebSite/project.json

```diff
@@ -4,9 +4,8 @@
         "kestrel": "Microsoft.AspNet.Hosting --server Kestrel --server.urls http://localhost:5000"
     },
     "dependencies": {
-        "Autofac": "3.3.0",
+        "Autofac.Framework.DependencyInjection": "4.0.0-*",
```

Comment:
Hmm... that might be good for the purpose of snapshotting. That is, when we push dev with this change to release/master it'll undoubtedly be broken...